Shared/Samples/Show labels on layer/ShowLabelsOnLayerView.swift

```diff
@@ -0,0 +1,111 @@
+                    try await featureTable.load()
+                    
+                    // Create a feature layer from the table.
+                    let featureLayer = FeatureLayer(featureTable: featureTable)
+                    
+                    // Add the layer to the map.
+                    map.addOperationalLayer(featureLayer)
+                    
+                    // Add labels to the layer.
+                    addLabels(to: featureLayer)
+                } catch {
+                    // Present an error if feature table fails to load.
+                    self.error = error
+                }
+            }
+            .alert(isPresented: $isShowingAlert, presentingError: error)
+    }
+}
+
+private extension ShowLabelsOnLayerView {
+    /// Add labels to a feature layer.
+    /// - Parameter layer: The `FeatureLayer` to add the labels to.
+    func addLabels(to layer: FeatureLayer) {
+        // Create label definitions for the two groups.
+        let democratDefinition = makeLabelDefinition(party: "Democrat", color: .blue)
+        let republicanDefinition = makeLabelDefinition(party: "Republican", color: .red)
+        
+        // Add the label definitions to the layer.
+        layer.addLabelDefinitions([democratDefinition, republicanDefinition])
+        
+        // Turn on labeling.
+        layer.labelsAreEnabled = true
+    }
+    
+    /// Create a label definition for the given PARTY field value and color.
+    /// - Parameters:
+    ///   - party: A `String` representing the party.
+    ///   - color: The `UIColor` for the label.
+    /// - Returns: A new `LabelDefinition` object.
+    private func makeLabelDefinition(party: String, color: UIColor) -> LabelDefinition {
+        // The styling for the label.
+        let textSymbol = TextSymbol(color: color, size: 12)
+        textSymbol.haloColor = .white
+        textSymbol.haloWidth = 2
+        
+        // An SQL WHERE statement for filtering the features this label applies to.
+        let whereStatement = "PARTY = '\(party)'"
+        
+        // An expression that specifies the content of the label using the table's attributes.
+        let expression = "$feature.NAME + ' (' + left($feature.PARTY,1) + ')\\nDistrict' + $feature.CDFIPS"
+        
+        // Make an arcade label expression.
+        let arcadeLabelExpression = ArcadeLabelExpression(arcadeString: expression)
+        let labelDefinition = LabelDefinition(labelExpression: arcadeLabelExpression, textSymbol: textSymbol)
+        labelDefinition.placement = .polygonAlwaysHorizontal
+        labelDefinition.whereClause = whereStatement
+        
+        return labelDefinition
+    }
+}
+
+private extension URL {
+    /// A feature table URL for a USA Congressional Districts Analysis.
+    static var usaCongressionalDistricts: URL {
+        URL(string: "https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/USA_Congressional_Districts_analysis/FeatureServer/0")!
+    }
+}
```

Comment:
Can you replace this raw URL with a portal item? https://arcgis.com/home/item.html?id=cc6a869374434bee9fefad45e291b779 Thanks.